src/app/loaders/express/helmet.ts

```diff
@@ -72,6 +73,9 @@ const helmetMiddlewares = () => {
       'https://www.gstatic.cn/',
       "'unsafe-inline'",
     ],
+    workerSrc: [
+      'blob:', // DataDog RUM session replay
```

Comment:
link to docs again

Reply:
https://github.com/opengovsg/FormSG/pull/3887/commits/ae7345811979a47b4be5f05b0eb1305f044362c7